hub/tasks/schedule.py

```diff
@@ -25,9 +100,16 @@ def load_auth_token():
 async def lifespan(app):
     """Schedule recurring tasks."""
     auth_token = load_auth_token()
+    run_scheduler = os.getenv("RUN_SCHEDULER", "false").lower() == "true"
```

Comment:
I think this could be READ_SCHEDULED_RUNS to match the others but more importantly so that it doesn't sound like it turns the whole scheduler on and off.
```suggestion
    run_scheduler = os.getenv("READ_SCHEDULED_RUNS", "false").lower() == "true"
```